src/app/opportunities-early-adopters/opportunities-early-adopters-page.component.scss

```
@@ -43,8 +43,19 @@
   }
 
   p {
-    font-size: 16px;
-    line-height: math.div(27, 16);
+    font-size: 22px;
+    line-height: math.div(27, 22);
+  }
+
+  h4 {
+    font-size: 24px;
+    font-weight: 700;
+    line-height: math.div(27, 22);
```

```suggestion
    line-height: math.div(27, 24);
```

Should be divided by `font-size`.